src/main/java/com/develhope/spring/vehicles/models/VehicleMapper.java

```diff
@@ -22,7 +23,7 @@ public class VehicleMapper {
      * @param vehicle l'entità Vehicle da convertire
      * @return il VehicleDTO convertito
      */
-    public VehicleCreatorDTO toDTO(Vehicle vehicle) {
+    public VehicleCreatorDTO toCreatorDTOFrom(Vehicle vehicle) {

```

Comment:
Cè incosistenza sul naming di questi metodi. A lorenzo non piace che si metta il "from" come ha fatto capire oggi in live. Perciò lasciamo solamente il "to" .